app/controllers/sign_up/completions_controller.rb

```diff
@@ -9,7 +9,7 @@ class CompletionsController < ApplicationController
 
     def show
       analytics.track_event(
-        Analytics::USER_REGISTRATION_AGENCY_HANDOFF_PAGE_VISIT,
+        'User registration: agency handoff visited',
```

Comment:
this should use new method, not the string:

```
analytics.user_registration_agency_handoff_page_visit()
```
or something

Reply:
updated